src/board/system76/common/power.c

```diff
@@ -317,8 +321,8 @@ static bool power_peci_limit(bool ac) {
 void power_set_limit(void) {
     static bool last_power_limit_ac = true;
     // We don't use power_state because the latency needs to be low
-#if EC_ESPI
-    if (!in_s0ix) {
+#if USE_S0IX
+    if (gpio_get(&SLP_S0_N)) {
```

Comment:
I'd leave this line as it was before, so:

```suggestion
    if (!in_s0ix) {
```

this fixes the LED blinking irregularly while Windows is sleeping by smoothing out the GPIO inputs.

Reply:
Reverted the change, there was a conflict when cherry-picking and I've chosen the incoming change.